app/views/layouts/application.html.erb

```diff
@@ -5,10 +5,19 @@
     <%= csrf_meta_tags %>
     <%= action_cable_meta_tag %>
 
-    <%= stylesheet_link_tag    'application', media: 'all' %>
+    <%= stylesheet_link_tag 'application', media: 'all' %>
   </head>
 
   <body>
+    <!-- TODO: Temporary auth links here, will be removed -->
+    <% if current_user %>
+      <%= link_to 'logout', destroy_user_session_path %>
+    <% else %>
+      <%= link_to 'Login', new_user_session_path %>
```

Comment:
what we are doing with translate or not now (other task)? 

I think we can add I18n.t for now, but without .yml files